components/display/waveshare_epaper.rst

```diff
@@ -75,6 +75,7 @@ Configuration variables:
   - ``2.13in`` (not tested)
   - ``2.13in-ttgo`` (T5_V2.3 tested)
   - ``2.13in-ttgo-b73`` (T5_V2.3 with B73 display tested)
+  - ``2.13in-ttgo-b73`` (All-in-one board. Don't use... it may kills epaper with current code)
```

Comment:
Can you are details about the board please? `All-in-one board` is a bit to generic.